docs/ARCHITECTURE.md

```diff
@@ -201,3 +201,67 @@ can be found under various directories such as `factory/`, `modprobe.d/`, `netwo
 `tools/`, `coccinelle/`, `.github/`, `.semaphore/`, `.mkosi/` host various
 utilities and scripts that are used by maintainers and developers. They are not
 shipped or installed.
+
+## UEFI
+The systemd project provides a UEFI bootloader, `systemd-boot`, and a stub that can be attached to a kernel
+image to allow loading it as a UEFI executable, `systemd-stub`. These components fully support UEFI Secure
```

Comment:
the linux kernel already contains a stub that allows it to run as uefi binary, we do not enable that, it can do that fine without systemd-stub (we used to be an *alternative* stub, but nowadays we chain load the kernel's own stub, because the "manual" boot protocol is kinda out of fashion).

just say that we "wrap a linux kernel in a PE binary, adding various features". or so